src/python/WMCore/MicroService/MSPileup/MSPileupTaskManager.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, msConfig, **kwargs):
         self.rucioAccount = msConfig.get('rucioAccount', 'wmcore_transferor')
         self.rucioUrl = msConfig['rucioUrl']  # aligned with MSCore init
         self.rucioAuthUrl = msConfig['rucioAuthUrl']  # aligned with MSCore init
+        self.cleanupTimeThreshold - msConfig.get('cleanupTimeThreshold', 15)
```

Comment:
typo. Please also add a comment saying that it's expected to be in days. Or maybe we should call it `cleanupDaysThreshold`? Up to you.

Reply:
done